src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicSpanExporter.java

```diff
@@ -130,6 +133,18 @@ public Builder commonAttributes(Attributes commonAttributes) {
       return this;
     }
 
+    /**
+     * Set a URI to override the default ingest endpoint.
```

Comment:
Probably nitpicky, but it feels wrong to call it an "endpoint" and then only use the scheme/host/port portion of the URI.  Maybe it clarifies things to call it a base URI or something similar?

Reply:
I'm trying to keep the same docs/naming as in the telemetry SDK (from which I copied the javadoc).

Reply:
aight